core/src/test/scala/org/broadinstitute/dsde/rawls/openam/MockUserInfoDirectives.scala

```diff
@@ -15,3 +15,10 @@ trait MockUserInfoDirectives extends UserInfoDirectives {
     provide(userInfo)
   }
 }
+
+trait MockAdminUserInfoDirectives extends UserInfoDirectives {
+  def requireUserInfo(magnet: ImplicitMagnet[ExecutionContext]): Directive1[UserInfo] = {
+    // just return the cookie text as the common name
+    provide(UserInfo("admin", OAuth2BearerToken("token"), 123, "123456789876543212345"))
+  }
+}
```

Comment:
nit: `MockAdminUserInfoDirectives`code is differently structured than `MockUserInfoDirectives` (including the comment); would be nice to have them exactly parallel. I don't feel strongly about this.